### ISSUE TITLE
No way to re-export a post that has been deleted from Wordpress

### ISSUE THREAD

**Issue by harrism** [Automattic/google-docs-add-on#81]
If you export a post to a Wordpress.org site using the add-on, and then go to the Wordpress wp-admin and delete that post, the add-on doesn't recognize that the post has been removed. So it still has an "update" button, but if you click "update" you get an error "unknown post".

I would like a way to export a document as a new draft even if it has been exported before (whether or not the previous exports were deleted).

**Comment by georgeh** [comment]
Thanks for using the add-on and going the extra mile by giving us a bunch of feedback! It really helps in determining the direction of the add-on.

Right now there's no way to change which post a document is linked to. The post id is stored as a property of the Google Doc. It's a UI challenge for the most part - what is a simple way to "link" or "unlink" a document to a post? Do you have a UI in mind?

For now, a work-around would be to create a copy of the document. The copy won't be linked to the post id, so it will create a new draft.

**Comment by harrism** [comment]
Hi George. I'm not sure about the UI. Perhaps you could just provide an "unlink" button, with requisite "are you sure?" warning pop-up?

**Comment by magnet365** [comment]
Hi, I'm experiencing the same issue that @harrism reported and wanted to comment on the question around the UI. A few things that would be helpful for me in this scenario would be:
- the "Unlink" button mentioned above, perhaps with more explicit text like "Disconnect from WordPress"
    - the are-you-sure confirmation would provide some nice text like "Your draft will still be available on WordPress.com"
- if the draft has been moved to the Trash in WordPress.com, detect that and show a notification in the Google Docs add-on; even better, provide a "Restore" button (i.e., don't show the "Update" button for trashed drafts); if permanently deleted on WordPress, provide a "Relink" button that generates a new draft post
- if your developers don't have the bandwidth right now, at least consider updating the error from "Error: Unknown post" to something the user will understand like "Error: We were unable to find and update your draft on WordPress.com. Your draft may have been permanently deleted. To continue syncing, create a new copy of this document and save it to WordPress."

**Comment by milhoan** [comment]
I agree, this would be a very useful addition.  Moving files around in my google docs folders has broken the connection between the existing posts and the docs.

**Comment by georgeh** [comment]
@magnet365 - I like the wording improvement suggestions! I'll see about adding those and if it's possible improve the functionality around deleted linked drafts.

**Comment by clayteller** [comment]
I'm experiencing this same issue. It would be great to see better handling of this in the add-on. Thanks

**Comment by NCARalph** [comment]
I've got the same problem. This is really pretty serious IMHO.
Is there any way to see the properties and manually change them?
The way many apps deal with this is Save and Save As, or perhaps Update and Save As.
Save As would also be useful rather than forcing the url page from the document name.

**Comment by rishirdua** [comment]
Wondering if there's been any update on the error message improvement?
